pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -65,7 +67,7 @@ in buildPythonPackage {
   '';
 
   postInstall = ''
-    ln -s ${sip}/${python.sitePackages}/PyQt5/* $out/${python.sitePackages}/PyQt5
+    ln -s ${sip}/${python.sitePackages}/PyQt5/sip.* $out/${python.sitePackages}/PyQt5/
```

Comment:
This change is based on the contents of the package at https://pypi.org/project/PyQt5-sip/ : it has only `PyQt5/sip.so` (and metadata files).